hiera.yaml

```diff
@@ -0,0 +1,16 @@
+---
+version: 5
+
+defaults:
+  datadir: 'data'
+  data_hash: yaml_data
+
+hierarchy:
+  - name: 'OS Major Release Overrides'
+    path: "family/%{facts.os.family}/%{facts.os.release.major}.yaml"
+  - name: 'Operating System'
+    path: "os/%{facts.os.name}.yaml"
+  - name: 'Operating System Family'
+    path: "family/%{facts.os.family}.yaml"
+  - name: 'Defaults'
+    path: 'defaults.yaml'
```

Comment:
This long hierarchy was taken from voxpupuli/puppet-gitlab for consistency.

Reply:
For consistency, would you be ok opening an additional PR to move the other parameter defaults to `defaults.yaml`?

Reply:
I can merge the latest changes into my branch, but would like to use common.yaml instead. We already do this for other modules.

Reply:
Sure, I will do his at work today.

Thanks!